### PR TITLE
Create user card and add tab views for card, history and rewards.

### PR DESCRIPTION
**Pull request by agalcalledjen**

### CODE REVIEW

File: client/js/screens/Card/Card.js [hunk 1]
```diff
@@ -1,22 +1,72 @@
...
+      index: 0,
+      routes: [
+        { key: 'first', title: 'Card', accessibilityLabel: 'Card Tab' },
+        { key: 'second', title: 'History', accessibilityLabel: 'History Tab' },
+        { key: 'third', title: 'Rewards', accessibilityLabel: 'Rewards Tab' }
+      ]
+    };
+  }
+
+  _renderLabel(props) {
```
Comment (siddharthparmar7) [inline client/js/screens/Card/Card.js]: if you are following this convention od putting _ in front of your methods then all of your methods should follow this convention (groupwide).

File: client/js/screens/Home/Home.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ const Home = props => {
     <View
       style={{ height: '100%', alignItems: 'center', justifyContent: 'center' }}
     >
-      <Text style={{ color: '#fff' }}>This is Home.</Text>
+      <Text style={{ color: 'yellow' }}>This is Home.</Text>
```
Comment (siddharthparmar7) [inline client/js/screens/Home/Home.js]: move your styles to your styles.js file

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
On the cards screen, remove all the warnings. Fix the bottom tab navigation icons, images cut off from the bottom tab navigation. Your Drawer throws an error. Please fix it.

Please make the requested changes. Thank you!

File: client/js/screens/Card/Card.js (hunk 1 above)
Comment (agalcalledjen) [reply client/js/screens/Card/Card.js]: Thanks for the tip and I have removed the underscore.

File: client/js/screens/Home/Home.js (hunk 2 above)
Comment (agalcalledjen) [reply client/js/screens/Home/Home.js]: These have been moved to styles.js.

**Comment by agalcalledjen** [comment]
Jen Y. had made changes to the bottom tab navigation icons in her latest pull request. Thanks!

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**